docs/product/components/page-titles.html

```diff
@@ -96,7 +96,7 @@ <h1>Page title</h1>
                     <h1>Page title</h1>
                     <p>Optional description de Finibus Bonorum et Malorum with an <a href="#">optional link</a>.</p>
                 </div>
-                <div>
+                <div class="md:mt8">
```

Comment:
@aaronshekey I don't love having part of the responsiveness in markup via this `.md:` class and part in the LESS file. I will try playing with padding on the parent `<div>`s but am open to other ideas.

Reply:
This is where I get hazy on these _needing_ to be a component. This can just as easily be mocked up as a pattern using atomic classes for full flexibility. There are lots of contexts for this, and may lead to fighting an overly-prescribed component.

We also need to account for the overall Questions, Tags, Users page headers yet—and if we don't include those, we'll likely need to explain why.